configs/bsp_sources.toml

```diff
@@ -1,16 +1,16 @@
 # BSP selection based on variant of filesystem
 [trixie]
-    bsp_version = "10.01.06"
+    bsp_version = "10.01.08.01"
```

Comment:
@jsuhaas22 If the plan is to stay inconsistent with Yocto ProcSDK versions, call it out as `10.01.08.02`
Ideally `10.01.06` should have been `10.01.06.01` earlier